software/contrib/pams.md

```diff
@@ -154,6 +156,8 @@ The submenu for each CV output has the following options:
 - `Quant` -- quantization scale
 - `Q Root` -- quantizer root: transposes the quantized output up by the number of semitones above C.
 - `Mute` -- if a channel is muted its output will always be zero. A muted channel can still be edited.
+- `Save` -- save the channel's current settings to one of 6 banks. Banks are shared across all channels
+- `Load` -- load a channel's settings from one of 6 banks. Banks are shared across all channels
 - `Reset` -- if set to `Y` all channel settings are reset to their defaults
```

Comment:
[optional] Reset actually has the options of `Cancel` and `OK` rather than `N` and `Y`. Even though this isn't technically part of this PR it might be worth adding it as it's such a small fix